UtopiaCity/Data/ApplicationDbContext.cs

```diff
@@ -82,5 +84,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         public DbSet<ArrivingPassenger> ArrivingPassengers { get; set; }
         public DbSet<ClinicVisit> ClinicVisit { get; set; }
         public DbSet<RealEstate> RealEstate { get; set; }
+        public DbSet<FireMessage> FireMessages { get; set; }
+        public DbSet<FireSafetyCheck> FireSafetyChecks { get; set; }
+        public DbSet<TransportManagement> TransportsManagements { get; set; }
```

Comment:
naming is ambigious. Also not consistent

Reply:
What namings do you suggest?